test/operators.jl

```diff
@@ -83,6 +83,16 @@ import Base.<
 
 @test isless('a','b')
 
+@testset "isgreater" begin
+    let isgreater = Base.isgreater
+        @test !isgreater(missing, 1)
+        @test  isgreater(5, 1)
+        @test !isgreater(1, 5)
+        @test  isgreater(1, missing)
+        @test  isgreater(1, NaN)
+    end
```

Comment:
`@testset` already creates a scope so we don't need `let`. Let's minimize the diff just in case we could convince @StefanKarpinski and export `isgreater` at some point :)

```suggestion
    isgreater = Base.isgreater
    @test !isgreater(missing, 1)
    @test  isgreater(5, 1)
    @test !isgreater(1, 5)
    @test  isgreater(1, missing)
    @test  isgreater(1, NaN)
```

Reply:
Thanks!

Reply:
Thanks!